packages/stripe-graphql/src/schema/payment-methods.ts

```diff
@@ -0,0 +1,64 @@
+import { builder } from '../builder'
+
+builder.objectType('StripePaymentMethods', {
+  fields: (t) => ({
+    object: t.exposeString('object'),
+    url: t.exposeString('url'),
+    hasMore: t.exposeBoolean('has_more'),
+    data: t.expose('data', {
+      type: ['StripePaymentMethod'],
+      nullable: false
+    })
+  })
+})
+
+// TODO: generic args for payment methods
+// export const paymentMethodsArgs = builder.args((t) => ({
+//   type: t({ // unable to get this to work
+//     type: PaymentMethodTypes,
+//     required: true,
+//     defaultValue: 'card'
+//   }),
+//   startingAfter: t.string({
+//     required: false
+//   }),
+//   endingBefore: t.string({
+//     required: false
+//   }),
+//   limit: t.int({
+//     required: false
+//   })
+// }))
+
+export const PaymentMethodTypes = builder.enumType('PaymentMethodTypes', {
```

Comment:
I would prefix this name with `Stripe` (`PaymentMethodTypes`) to keep things namespaced